reana_client/validation.py

```diff
@@ -521,6 +521,29 @@ def _validate_cwl_parameters(reana_yaml):
         )
         sys.exit(1)
 
+    def _check_dangerous_operations(workflow):
+        """Check for "baseCommand" and "arguments" in workflow.
+
+        If these keys are found, validate if they have dangerous operations.
+        """
+        for k, v in workflow.items():
+            if k in dang_keys:
+                if isinstance(v, list):
+                    for val in v:
+                        _validate_dangerous_operations(val, None)
```

Comment:
The idea here is to pass the step if it exists. Maybe we can pass `workflow.get('id')`, at least we would have some useful info for the user.

Reply:
What would be the name of the `id` in the msg. `step`/`id` or something else?

Reply:
If we pass the `id` from here, then we need the flag in `_validate_dangerous_operations` to check if it's coming from cwl to show the output msg accordingly.

Reply:
We can just treat it as the step, in the end is kind of similar.